code/style.css

```diff
@@ -138,3 +140,16 @@ button:hover {
   opacity: 0.9;
   transition: all 0.2s ease;
 }
+
+footer {
+  position: fixed;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  height: 100px;
+  background-color: #6f69ac;
+  color: white;
+  text-align: center;
+  font-family: 'Kranky', cursive;
+  padding-top: 25px;
+}
```

Comment:
Because of the position:fixed on your footer Kara wasn't able to try out your bot on her laptop because the footer was on top of the chat. If you placed it below with z-index it would be placed underneath the chat on smaller screens.